### PR TITLE
Zabbix7

### PR DESCRIPTION
**Pull request by mimugmail**
Add pkg's for plugins

### CODE REVIEW

File: config/24.1/plugins.conf [hunk 1]
```diff
@@ -25,9 +25,11 @@ net-mgmt/telegraf				arm
 net-mgmt/zabbix-agent@zabbix5			arm
 net-mgmt/zabbix-agent@zabbix6			arm
 net-mgmt/zabbix-agent@zabbix64			arm
+net-mgmt/zabbix-agent@zabbix7			arm
```
Comment (fichtner) [inline config/24.1/plugins.conf:28]: is there a PR coming? adding these to 24.1 and the packages to 24.7 is going to cause friction ;)

File: config/24.7/ports.conf [hunk 2]
```diff
@@ -102,6 +102,8 @@ net-mgmt/zabbix6-agent				arm
 net-mgmt/zabbix6-proxy				arm
 net-mgmt/zabbix64-agent				arm
 net-mgmt/zabbix64-proxy				arm
+net-mgmt/zabbix7-agent				arm
```
Comment (fichtner) [inline config/24.7/ports.conf:105]: ideally add the packages to both 24.1 and 24.7

**Comment by fichtner** [comment]
@mimugmail this needs fixing first ;)

**Comment by mimugmail** [comment]
Wow, never ever create a PR via mobile :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**